polymer-cli build broken with npm install

No problem with yarn, but under npm:

> [14:09:11] Starting 'build'...
> src/build/optimize-streams.ts(189,34): error TS2345: Argument of type 'JSDefaultCompileTransform' is not assignable to parameter of type 'ReadWriteStream'.
>   Types of property 'pause' are incompatible.
>     Type '() => Readable' is not assignable to type '() => ReadWriteStream'.
>       Type 'Readable' is not assignable to type 'ReadWriteStream'.
>         Types of property 'pause' are incompatible.
>           Type '() => Readable' is not assignable to type '() => ReadWriteStream'.
> events.js:163
>       throw er; // Unhandled 'error' event
>

--- a/package.json
+++ b/package.json
@@ -83,7 +83,7 @@
   },
   "devDependencies": {
     "@types/fs-extra": "0.0.37",
-    "@types/node": "^6",
+    "@types/node": "=6.0.65",
     "@polymer/tools-common": "^1.0.1",
     "chai": "^3.5.0",
     "clang-format": "=1.0.48",

